cmd/queue/main.go

```diff
@@ -137,6 +142,10 @@ type config struct {
 	ServingRequestMetricsBackend string `split_words:"true" required:"true"`
 	ServingRequestLogTemplate    string `split_words:"true" required:"true"`
 	ServingReadinessProbe        string `split_words:"true" required:"true"`
+	TracingConfigEnable          string `split_words:"true" required:"true" default:"false"`
+	TracingConfigZipkinEndpoint  string `split_words:"true" required:"true"`
+	TracingConfigDebug           string `split_words:"true" required:"true" default:"false"`
+	TracingConfigSampleRate      string `split_words:"true" required:"true" default:"0"`
```

Comment:
If you change to the correct types (bool/float64) the defaults will be implicitly correct.

Reply:
just tried this out -- you're correct.  Thanks!